docs/user/filesystem.md

```diff
@@ -502,3 +439,59 @@ $ D_IL_REPORT=-1 LD_PRELOAD=/usr/lib64/libioil.so dd if=/dev/zero of=./bar bs=1G
     function to close stderr on exit, so for many basic commands such as cp and cat
     whilst the interception library will work it is not possible to see the summary
     generated by the interception library.
+
+### Advanced Usage
+
+DFuse will only create one kernel level mount point regardless of how it is
+launched. How POSIX containers are represented within that mount point varies
+depending on the DFuse command-line options. In addition to mounting a single
+POSIX container, DFuse can also operate in two other modes detailed below.
+
+#### Pool Mode
+
+If a pool uuid is specified but not a container uuid, then the containers can be
+accessed by the path `<mount point>/<container uuid>`. The container uuid
+will have to be provided from an external source.
+
+```bash
+$ daos cont create tank --label mycont2 --type POSIX
+  Container UUID : 0db21789-5372-4f2a-b7bc-14c0a5e968df
+  Container Label: mycont2
+  Container Type : POSIX
+
+Successfully created container 0db21789-5372-4f2a-b7bc-14c0a5e968df
+$ dfuse -m /tmp/dfuse --pool tank
+$ ls -l /tmp/dfuse/
+ls: cannot open directory '/tmp/dfuse/': Operation not supported
+$ ls -l /tmp/dfuse/0db21789-5372-4f2a-b7bc-14c0a5e968df
+total 0
+$ ls -l /tmp/dfuse/8a8f08bb-5034-41e8-b7ae-0cdce347c558
```

Comment:
Creation of this pool seems to be missing in this new documentation.

Reply:
Fixed in new version.